frontend/src/components/ShootWorkers/WorkerGroup.vue

```diff
@@ -76,9 +76,16 @@ export default {
           })
           if (machineType.storage) {
             description.push({
-              title: 'Volume Size',
-              value: `${machineType.storage.size}`
+              title: 'Volume Type',
```

Comment:
I do not really understand what has priority from looking at the code. I looks like that double Volume Type and Volume Size descriptions can occur. It would split this large function into many smaller function for each description type.

Reply:
This should not happen as there should always be either storage **or** volume type defined in the cloud profile. However, I will try to improve the code, make it more robust and readable.